mapping/sustainability.yaml

```
@@ -954,7 +954,7 @@
     module: Climate finance
     indicator: Anti-corruption impact
     disclosure format: Identify the entities acting as independent monitors of the project (E.g. \[free text\])
-    mapping: 'Project level: Add an entry to `parties` with ''independentMonitor'' included in its `.roles`.'
+    mapping: 'Project level: [Add an organization](../common.md#add-an-organization) for the independent monitor and add ''independentMonitor'' to its `.roles` array.'
```

When a CoST IDS element occurs in more than one module, in order to maintain a single source of truth, let's only document the mapping in one place. Likewise, please remove the example from this occurrence so that we don't need to keep it up to date in two places.

This change will cause a build error until #428 is merged and 0.9-dev is merged into this branch.

```suggestion
    mapping: 'See [independent monitoring (institutional module)](institutional-independent-monitoring).'
```

---

> This change will cause a build error until https://github.com/open-contracting/infrastructure/pull/428 is merged and 0.9-dev is merged into this branch.

Done so this change shouldn't cause any issues.

---

Thanks @duncandewhurst Changes made.